quickstart/deploy-mayastor.md

```diff
@@ -6,317 +6,99 @@
-kubectl apply -f https://raw.githubusercontent.com/openebs/mayastor-control-plane/v1.0.2/deploy/rest-service.yaml
-```
-{% endtab %}
-
-{% tab title="Example Output" %}
-```text
-deployment.apps/rest created
-service/rest created
-```
-{% endtab %}
-{% endtabs %}
-
-Verify that the REST pod is running.
+## Installation via helm
 
+1. Add the `mayastor-extensions` helm repository.
 {% tabs %}
 {% tab title="Command" %}
 ```text
-kubectl get pods -n mayastor --selector=app=rest
+helm repo add mayastor https://openebs.github.io/mayastor-extensions/ 
 ```
 {% endtab %}
-
-{% tab title="Example Output" %}
+{% tab title="Output" %}
 ```text
-NAME                    READY   STATUS    RESTARTS   AGE
-rest-5cd9665499-cdgmm   1/1     Running   0          2m35s
+"mayastor" has been added to your repositories
 ```
 {% endtab %}
 {% endtabs %}
 
-### CSI Controller
 
-
-{% tabs %}
-{% tab title="Command \(GitHub Latest\)" %}
-```text
-kubectl apply -f https://raw.githubusercontent.com/openebs/mayastor-control-plane/v1.0.2/deploy/csi-deployment.yaml
-```
-{% endtab %}
-{% endtabs %}
-
-Verify that the CSI-controller pod is running.
+Run the following command to discover all the _stable versions_ of the added chart repository:
 
 {% tabs %}
 {% tab title="Command" %}
 ```text
-kubectl get pods -n mayastor --selector=app=csi-controller
+helm search repo mayastor --versions
 ```
 {% endtab %}
-
-{% tab title="Example Output" %}
+{% tab title="Sample Output" %}
 ```text
-NAME                             READY   STATUS    RESTARTS   AGE
-csi-controller-579f77f64-7dq7g   3/3     Running   0          39m
+ NAME             	CHART VERSION	APP VERSION  	DESCRIPTION                       
+mayastor/mayastor	2.0.0        	2.0.0       	Mayastor Helm chart for Kubernetes
 ```
 {% endtab %}
 {% endtabs %}
 
-### MSP Operator
- 
-{% tabs %}
-{% tab title="Command \(GitHub Latest\)" %}
-```text
-kubectl apply -f https://raw.githubusercontent.com/openebs/mayastor-control-plane/v1.0.2/deploy/msp-deployment.yaml
-```
-{% endtab %}
-{% endtabs %}
+{% hint style="info" %}
+To discover all the versions (including unstable versions), execute:
+`helm search repo mayastor --devel --versions`
+{% endhint %}
 
-Verify that the pool operator pod is running.
 
+3. Run the following command to install Mayastor _version 2.0_.
 {% tabs %}
 {% tab title="Command" %}
 ```text
-kubectl get pods -n mayastor --selector=app=msp-operator
-```
-{% endtab %}
-
-{% tab title="Example Output" %}
-```text
-NAME                            READY   STATUS    RESTARTS   AGE
-msp-operator-7849d59fcd-mcw5b   1/1     Running   0          21s
-```
-{% endtab %}
-{% endtabs %}
-
-
-### Data Plane
-
-{% tabs %}
-{% tab title="Command \(GitHub Latest\)" %}
-```text
-kubectl apply -f https://raw.githubusercontent.com/openebs/mayastor/master/v1.0.2/mayastor-daemonset.yaml
+helm install mayastor mayastor/mayastor -n mayastor --create-namespace
```

Comment:
I think we might need to be more specific. As given, I think the command installs the latest version of the chart, *whatever* version of Mayastor that might describe. It works whilst 2.0 (or 2.0.x) is the latest version of the Helm chart.  However, once we release v2.1, or v3.0 (or even v1.03) that won't be correct.

Reply:
Got it..I'll be updating it.

Reply:
The thing that I'll have to keep a note of is: with every new patch this command must be updated.